js/index.js

```diff
@@ -64,4 +66,48 @@ const messageList = messageSection.querySelector('ul');
     messageList.appendChild(newMessage);
     messageSection.hidden = false;
     messageForm.item(0).reset();
-});
\ No newline at end of file
+});
+
+const githubRequest = new XMLHttpRequest();
+githubRequest.open('GET', 'https://api.github.com/users/raissanfg/repos');
+githubRequest.send();
+githubRequest.addEventListener('load', function() {
+const repositories = JSON.parse(this.response)
+
+const projectSection = document.querySelector('#projects')
+const projectList = projectSection.querySelector('ul')
+
+ for (let i=0; i < repositories.length; i++) {
+    var project = document.createElement('li');
+    var repositoryLink = document.createElement('a');
+    repositoryLink.href = repositories[i].html_url;
+    repositoryLink.innerHTML = repositories[i].name;
+    projectList.appendChild(project);
+    project.appendChild(repositoryLink);
+
+ }
+});
+
+function renderProjectsWithFetch() {
+    fetch('http://api.github.com/user/raissanfg/repos')
+    .then((res) => res.json())
+    .then((data) => {
+
+    const repositories = JSON.parse(this.response)
+    const projectSection = document.querySelector('#projects')
+    const projectList = projectSection.querySelector('ul')
+
+         for (let i=0; i < repositories.length; i++) {
+         var project = document.createElement('li' );
+         var repositoryLink = document.createElement('a');
+         repositoryLink.href = repositories[i].html_url;
+         repositoryLink.innerHTML = repositories[i].name;
+         projectList.appendChild(project);
+         project.appendChild(repositoryLink);
+         proja.setAttribute('id', 'proja')
```

Comment:
I know I previously suggested setting an ID attribute so you could style these project links; but since you've now added a `projects ul li {...}` section (line 165 in your css) to your styles, this won't be needed anymore. Also it currently would cause an error because you're trying to set an attribute on a variable called `proja`, which hasn't been declared anywhere. 

You can just remove this line now, it's no longer needed